pom.xml

```diff
@@ -34,7 +34,8 @@
         <module>security/jwt</module>
         <module>security/jwt-cookie</module>
         <module>security/keycloak</module>
-        <module>security/keycloak-authz</module>
+        <!-- TODO: disabled keycloak-authz https://github.com/quarkusio/quarkus/issues/17222 -->
+        <!-- <module>security/keycloak-authz</module> -->
```

Comment:
We don't need to exclude the whole module, but just adding the `@Authenticated` annotation to the resources.
Also, this issue is duplicated, the right issue is https://github.com/quarkusio/quarkus/issues/17164